internal/backend/svc/banner.txt

```diff
@@ -1,13 +1,13 @@
 
-        _..--~~~~--.._             autokitteh    {{ .Version }}
-     _--              --_          PID:          {{ .PID }}
-    '                    `         gRPC/HTTP:    {{ .Addr }}
-   '                      `        Web{{ .WebVer }}  {{ .WebAddr }}
-  '                        `       {{ .Mode }}
- |                    |\   |       
- |                    | \  |       {{ .Temporal }}
- |          .,    _,-~   \ |       {{ .TemporalUI }}
-  \         \ `.-'        \/ 
+        _..--~~~~--.._             autokitteh     {{ .Version }}
+     _--              --_          PID:           {{ .PID }}
+    '                    `         gRPC/HTTP API: {{ .Addr }}
+   '                      `        Web{{ .WebVer }}   {{ .WebAddr }}
+  '                        `       Aux:           {{ .AuxAddr }}
```

Comment:
99% of users: "WTF is Aux? Is that a new protocol I never heard of? Oh, it's a web UI/API which isn't *the* web UI, which uses HTTP but isn't on *the* HTTP port?"